include/vkgcDefs.h

```diff
@@ -503,6 +504,8 @@ struct PipelineOptions {
   bool internalRtShaders;                         ///< Whether this pipeline has internal raytracing shaders
   unsigned forceNonUniformResourceIndexStageMask; ///< Mask of the stage to force using non-uniform resource index.
   bool reserved16;
+  bool explicitColorExport; ///< Explicitly build color export shader, UnlinkedStageFragment elf will
+                            ///  return extra meta data.
```

Comment:
I think this fits better in `GraphicsPipelineBuildInfo`. It's not really a tuning option, and it fits next to `enableUberFetchShader` and `unlinked`. Maybe call it `enableColorExportShader`?